controller_interface/src/controller_interface_base.cpp

```diff
@@ -152,16 +151,32 @@ return_type ControllerInterfaceBase::init(
     [this](const rclcpp_lifecycle::State & previous_state) -> CallbackReturn
     {
       this->stop_async_handler_thread();
+      lifecycle_id_.store(this->get_lifecycle_state().id(), std::memory_order_release);
       auto transition_state_status = on_error(previous_state);
       this->release_interfaces();
       return transition_state_status;
     });
 
+  switch (on_init())
+  {
+    case LifecycleNodeInterface::CallbackReturn::SUCCESS:
+      break;
+    case LifecycleNodeInterface::CallbackReturn::ERROR:
+    case LifecycleNodeInterface::CallbackReturn::FAILURE:
+      RCLCPP_DEBUG(
```

Comment:
Does this frequently happen? Why is this a DEBUG log?

Reply:
Good question. I just moved the block down and didn't touch anything

Reply:
```suggestion
      RCLCPP_ERROR(
```